packages/template-retail-react-app/app/commerce-api/einstein.js

```diff
@@ -53,6 +53,52 @@ class EinsteinAPI {
         return body
     }
 
+    /**
+     * Given a product source, returns the data that Einstein requires
+     */
+    _destructureProduct(product) {
```

Comment:
It seems to be that we extract a field from a product, and add a few more fields to fit what Einstein requires, construct/create/get sounds fitter than destructure to me 🤔 
```suggestion
            product: this._constructEinsteinProduct(product),
```

Reply:
This change has been applied.